backend/src/main/java/org/eclipse/tractusx/puris/backend/stock/controller/ItemStockRequestApiController.java

```diff
@@ -62,6 +61,32 @@ public class ItemStockRequestApiController {
     private Validator validator;
     private final Pattern bpnlPattern = PatternStore.BPNL_PATTERN;
 
+    private final Pattern urnPattern = PatternStore.URN_OR_UUID_PATTERN;
+
+
+    @Operation(summary = "This endpoint receives the ItemStock 2.0.0 requests")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Ok"),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    @GetMapping("request/{materialnumber}/{direction}")
+    public ResponseEntity<ItemStockSamm> getMappingItemStock2(@RequestHeader("edc-bpn") String bpnl,
```

Comment:
Please add representation to the end to append /$value and return 501 else (same as for PartTypeInformation). Please consider during workflow when performing the proxy call